lib/Infer/EnumerativeSynthesis.cpp

```diff
@@ -193,7 +209,7 @@ void getGuesses(std::vector<Inst *> &Guesses,
 
   std::vector<Inst *> PartialGuesses;
 
-  std::vector<Inst *> Comps(Inputs.begin(), Inputs.end());
+  std::deque<Inst *> Comps(Inputs.begin(), Inputs.end());
```

Comment:
dequeues are pretty slow, you're sure you want to do this?

Reply:
if you're sorting anyway, why do you need to add guesses to the front?

Reply:
No, can leave it as a vector for now.
This will have to change eventually though.